lib/modules/manager/bazel/rules/index.ts

```diff
@@ -30,7 +30,7 @@ const supportedRules = [
   ...httpRules,
   ...mavenRules,
 ];
-export const supportedRulesRegex = regEx(`^${supportedRules.join('|')}$`);
+export const supportedRulesRegex = regEx(`^(?:${supportedRules.join('|')})$`);
```

Comment:
why this change?

Reply:
Apologies, I forgot to mention this change in the description—I'll add something now. It's a small bug fix because `^a|b$` is interpreted as `(^a)|(b$)` rather than `^(a|b)$`. The use of `(?:)` rather than `()` just means that we use a non-matching group, which will likely be slightly more efficient as the regex engine will not need to record where the match happened.